### DISCUSSION TITLE
Manual way to remove a Podcast URL that is now a 404 ?

### DISCUSSION THREAD

**Discussion by Nightdive99** [thrillfall/nextcloud-gpodder#108] [Q&A] [answered]
I have a Podcast feed that is now a 404.
My clients (Antennapod and KATS) always throw an error if I manually sync in the client and they reject the broken podcast url.
That means I can't add+remove the podcast in the client and sync that back to the server.

I tried to redirect the broken URL to my http server to a valid rss file but the clients are of course unhappy with the incorrect ssl certificate.
Removing gpoddersync from nextcloud doesn't remove the database....

Does anyone have an idea how I could get rid this broken podcast URL ?

**Comment by JonOfUs** [comment, marked as answer]
Currently there is no way to reset the data apart from manually altering the database.

You could talk to the API directly to mark the 404 feed as removed. This way, it should no longer be synchronized to Kasts and Antennapod. But for this it's easiest to use the terminal/command line.
The command for this is:
`curl -u username:password -H "Content-Type: application/json" -X POST -d '{"add": [], "remove": ["https://example.com/feed.rss"]}' https://cloud.example.com/index.php/apps/gpoddersync/subscription_change/create`
Just replace username&password with your Nextcloud login data, replace `https://example.com/feed.rss` with the problematic feed URL and `cloud.example.com` with your Nextcloud's domain.
This command should work the same on Linux, Windows and Mac.

**Reply by Nightdive99** [reply]
Thank you very much for the working solution !
I tried nearly the same before but I used my general Account password and not an application specfic one and that didn't work.
I fixed my problem with your hint !
